charts/dapr/charts/dapr_scheduler/templates/dapr_scheduler_statefulset.yaml

```diff
@@ -10,7 +10,11 @@ metadata:
     {{ $key }}: {{ tpl $value $ }}
     {{- end }}
 spec:
+  {{- if eq .Values.global.ha.enabled true }}
+  replicas: {{ .Values.global.ha.replicaCount }}
```

Comment:
Note that `global.ha.replicaCount` can be any number. For a service based on etcd you MUST have an odd number such as 1, 3, or 5 (no 2, 4, or a number greater than 5)

Even for Placement `ha.replicaCount` isn't used. Docs say:

> Number of replicas of control plane services in Highly Availability mode
> Note that in HA mode, Dapr Placement has 3 replicas and that cannot be configured.

Reply:
good point. I'll update to have odd numbers only

Reply:
@ItalyPaleAle please check when you have time